go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/skylight-hq/phinvads-go
 
-go 1.23.0
+go 1.23.1
 
 require (
+	github.com/CDCgov/phinvads-fhir v0.0.0-20240925171804-b61370f4cab1
```

Comment:
I think this is just a leftover from being in the other repo and it doesn't look like we need it. 

If you run `go mod tidy` it should clean up both `go.mod` and `go.sum`.